packages/select/src/FilterMultiSelect/components/ListBoxSection/ListBoxSection.tsx

```diff
@@ -1,24 +1,54 @@
-import React from "react"
+import React, { ReactNode } from "react"
+import { v4 } from "uuid"
 import { MultiSelectItem } from "../../../types"
 import styles from "./ListBoxSection.module.scss"
-
 export interface ListBoxSectionProps {
   items: MultiSelectItem[]
   /**
    * Becomes an aria-label on the section, informing
    * unsighted users
    */
   sectionName: string
+  /** If provided, will override the aria-label for this group */
+  sectionHeader?: ReactNode
   children: (item: MultiSelectItem) => React.ReactNode
 }
 
 export const ListBoxSection = ({
   items,
   sectionName,
   children,
-}: ListBoxSectionProps): JSX.Element => (
-  <div aria-label={sectionName} className={styles.listBoxSection}>
-    {Array.from(items).map(node => node != undefined && children(node))}
-  </div>
-)
+  sectionHeader,
+}: ListBoxSectionProps): JSX.Element => {
+  const listSectionId = sectionHeader ? v4() : undefined
+  return (
+    <li role="presentation">
+      <ul
+        className={styles.listBoxSection}
+        aria-label={sectionName}
+        aria-labelledby={sectionHeader ? listSectionId : undefined}
+        aria-describedby={listSectionId || undefined}
+        role="group"
+      >
+        {sectionHeader && (
+          <li
+            className={styles.listBoxSectionHeader}
+            id={listSectionId}
+            role="presentation"
+          >
+            <span className="sr-only">{sectionName}. </span>
```

Comment:
I'm proposing we remove this, but where does this global CSS class even come from? Don't forget we have the `VisuallyHidden` component from `@kaizen/a11y` for this.